other/bootstrap_daemon/docker/Dockerfile

```diff
@@ -11,42 +14,40 @@ RUN apt-get update && apt-get install -y \
       git \
       yasm \
       libsodium-dev \
+      python3" && \
+    export RUNTIME_PACKAGES="\
       libconfig-dev \
-      python3 \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-# install toxcore and daemon
-WORKDIR /root/
-RUN git clone https://github.com/irungentoo/toxcore
-WORKDIR /root/toxcore/
-RUN ./autogen.sh
-RUN ./configure --enable-daemon
-RUN make -j`nproc`
-RUN make install -j`nproc`
-RUN ldconfig
-
-WORKDIR /root/toxcore/other/bootstrap_daemon/
-
-# add new user
-RUN useradd --home-dir /var/lib/tox-bootstrapd --create-home \
+      libsodium13" && \
+    apt-get update && apt-get install -y $BUILD_PACKAGES $RUNTIME_PACKAGES && \
+    git clone https://github.com/irungentoo/toxcore && \
+    cd toxcore && \
+    ./autogen.sh && \
+    ./configure --enable-daemon && \
+    make -j`nproc` && \
+    make install -j`nproc` && \
+    ldconfig && \
+    useradd --home-dir /var/lib/tox-bootstrapd --create-home \
       --system --shell /sbin/nologin \
       --comment "Account to run Tox's DHT bootstrap daemon" \
-      --user-group tox-bootstrapd
-RUN chmod 700 /var/lib/tox-bootstrapd
-
-RUN cp tox-bootstrapd.conf /etc/tox-bootstrapd.conf
-
-# remove all the example bootstrap nodes from the config file
-RUN N=-1 && \
+      --user-group tox-bootstrapd && \
+    chmod 700 /var/lib/tox-bootstrapd && \
+    cp other/bootstrap_daemon/tox-bootstrapd.conf /etc/tox-bootstrapd.conf && \
+    N=-1 && \
     while grep -q "bootstrap_nodes =" /etc/tox-bootstrapd.conf; \
     do \
       head -n $N tox-bootstrapd.conf > /etc/tox-bootstrapd.conf; \
```

Comment:
This should be `other/bootstrap_daemon/tox-bootstrapd.conf` instead of `tox-bootstrapd.conf`, as you have removed `WORKDIR /root/toxcore/other/bootstrap_daemon/`.